homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -111,9 +113,15 @@ def state(self):
         # key Fuel_type is not available when the fuel station is closed, use "get" instead of "[]" to avoid exceptions
         return self._coordinator.data[self._station_id].get(self._fuel_type)
 
+    @property
+    def unique_id(self) -> str:
```

Comment:
This is a good idea, could you split this part (unique id) into a separate PR? I'm not sure that hiding the latitude and longitude from the attributes is the right way to go, and having two different PRs (for two different logical changes) would allow them to be handled separately.